omniscidb/ArrowStorage/ArrowStorageUtils.cpp

```diff
@@ -1082,6 +1089,9 @@ std::shared_ptr<arrow::DataType> getArrowImportType(hdk::ir::Context& ctx,
         return list(getArrowImportType(ctx, elem_type));
       }
     }
+    case hdk::ir::Type::kNull:
+      throw std::runtime_error(
+          "Null should be converted to ArrowType according to getTargetImportType(...).");
```

Comment:
Runtime errors are for users and should explain to users what happened. This is not a useful error message, the previously used message about unsupported type is more informative.